ibc.ts/src/scenario/runChains.ts

```diff
@@ -0,0 +1,175 @@
+import { promises as fs } from "fs";
+import {
+    spawn,
+    ChildProcess,
+    ChildProcessWithoutNullStreams
+} from "child_process";
+import * as readline from "readline";
+import Debug from "debug";
+import { SDK } from "codechain-sdk";
+import { delay } from "../common/util";
+import { H256, PlatformAddressValue } from "codechain-primitives/lib";
+
+const debug = Debug("runChains");
+
+async function main() {
+    console.log("Build foundry");
+    await buildFoundry();
+    console.log("Reset DB");
+    await resetDB();
+    console.log("Run Chain A");
+    await runChainA();
+    console.log("Run Chain B");
+    await runChainB();
+    console.log("Check the chains");
+    await checkChainAAndBAreRunning();
+    console.log("Chains are running!");
+}
+
+main().catch(console.error);
+
+async function buildFoundry() {
+    const cargoProcess = spawn("cargo", ["build"], {
+        cwd: "../"
+    });
+    streamOutToDebug(cargoProcess);
+    await waitForChildProcess(cargoProcess);
+}
+
+async function resetDB() {
+    debug("Remove chain A DB");
+    await fs.rmdir("./chainA/db", {
+        recursive: true
+    });
+
+    debug("Remove chain B DB");
+    await fs.rmdir("./chainB/db", {
+        recursive: true
+    });
+}
+
+async function runChainA() {
+    const foundryProcess = spawn(
+        "../../target/debug/foundry",
+        ["-c", "./chainA.schem.json", "--config", "./chainA.config.toml"],
+        {
+            cwd: "./chainA"
+        }
+    );
+    streamOutToDebug(foundryProcess);
+}
+
+async function runChainB() {
+    const foundryProcess = spawn(
+        "../../target/debug/foundry",
+        ["-c", "./chainB.schem.json", "--config", "./chainB.config.toml"],
+        {
+            cwd: "./chainB"
+        }
+    );
+    streamOutToDebug(foundryProcess);
+}
+
+async function checkChainAAndBAreRunning() {
+    // Wait for Foundry to listen on the port, three seconds is a random value.
```

Comment:
a random value -> just an arbitrary value?

Reply:
Good. I'll fix it to "Wait for Foundry to listen on the port; three seconds is an arbitrary value."

Reply:
@junha1 I fixed it.